com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -127,6 +127,18 @@ private enum State
             Connected,
         }
 
+        /// <summary>
+        /// When setting relay server data, it asks for a string parameter that can only be one of the below values.
+        /// These are taken from the relay error message when setting the wrong string
+        /// </summary>
+        public static class RelayConnectionTypes
+        {
+            public const string UDP = "udp";
+            public const string DTLS = "dtls";
+            public const string WS = "ws";
+            public const string WSS = "wss";
```

Comment:
They would be used by users. Right now, they have to do this
`utp.SetRelayServerData(new RelayServerData(hostAllocation, "wss"));`
With the hardcoded string. With the above, it'd help users not have magic strings laying around.

Reply:
It is problematic to duplicate the strings. If/When, in the future the "dtls" string changes, there will be two places to update, unless the Relay server somehow uses the string above.

It's minor, it's no big deal, but I wanted to call it out.

Reply:
The ideal case would be to have those strings come from the Relay SDK. If those strings change, that'd be considered a breaking change on their server's API, but it could still happen, I agree. I'll check with folks in the relay channels, see if I can get some traction there.
While looking for those strings in the Relay SDK, I also notice RelayServerData has `supportedConnectionTypes` which has those hard coded strings I'd need to replace there as well. (And no, those strings are not defined in any SDKs that I can see)